AIAgent/run_training.py

```diff
@@ -69,107 +79,101 @@ def next(self):
         self.total_epochs += 1
 
 
-def model_saver(
-    epochs_info: EpochsInfo, svm_name: str, dir: Path, model: torch.nn.Module
-):
+def model_saver(epochs_info: EpochsInfo, dir: Path, model: torch.nn.Module):
     """
     Use it to save your torch model with some info in the following format: "{`total_epochs` + `epoch`}_{`svm_name`}" in `dir` directory
 
     Parameters
     ----------
     :param EpochsInfo `epochs_info`: EpochsInfo's instance
-    :param str `svm_name`: name of svm
     :param torch.nn.Module `model`: model to save
     :param Path `dir`: directory
 
     """
-    path_to_model = os.path.join(dir, f"{epochs_info.total_epochs}_{svm_name}")
+    path_to_model = os.path.join(dir, f"{epochs_info.total_epochs}")
     torch.save(model.state_dict(), Path(path_to_model))
 
 
 def run_training(
     svm_configs: list[SVMConfig],
     datasets_of_platforms: dict[PlatformName, DatasetConfig],
-    n_startup_trials: int,
-    n_trials: int,
+    optuna_config: OptunaConfig,
+    training_config: TrainingConfig,
     path_to_weights: Optional[Path],
     logfile_base_name: str,
     statistics_collector: StatisticsCollector,
 ):
     models_path = os.path.join(TRAINED_MODELS_PATH, logfile_base_name)
     os.makedirs(models_path)
 
-    sampler = optuna.samplers.TPESampler(n_startup_trials=n_startup_trials)
+    sampler = optuna.samplers.TPESampler(
+        n_startup_trials=optuna_config.n_startup_trials
+    )
     study = optuna.create_study(
-        sampler=sampler, direction=TrainingConfig.STUDY_DIRECTION
+        sampler=sampler, direction=optuna_config.study_direction
     )
     epochs_info = EpochsInfo()
+
+    maps: List[GameMap] = list()
+    server_count = 0
     for svm_config in svm_configs:
```

Comment:
вот это когда-нибудь надо будет менять, когда будем поддерживать другие платформы. чтобы это было очевидно, можно написать 

```suggestion
    server_count = 0
    for svm_config in svm_configs:
        if svm_config.platform_name != "dotnet":
            continue
```

Reply:
Я вообще что-то запуталась в конфигах svm, платформ и т. д. Я думаю, нужно это обсудить